src/Sulu/Bundle/MediaBundle/Media/Storage/GoogleCloudStorage.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\Storage;
+
+use League\Flysystem\Filesystem;
+use League\Flysystem\FilesystemInterface;
+use Superbalist\Flysystem\GoogleStorage\GoogleStorageAdapter;
+
+class GoogleCloudStorage extends FlysystemStorage
```

Comment:
Why do we need a special class for Google cloud? I don't see any code in here, which couldn't be used in the FlysystemStorage` class as well, assuming that FlySystem will only be used with clouds.

Reply:
The `getPath` of this class is bound to the storage (`getUrl` is not part of the `AdapterInterface`) For AWS we will implement this function in a different way.